tests/testthat/_snaps/Model-validity.new.md

```diff
@@ -0,0 +1,133 @@
+# v_model_logistic_kadane_beta_gamma returns message for non-scalars
```

Comment:
same for these non-plot snapshots actually

Reply:
As above